aten/src/ATen/native/cpu/Activation.cpp

```diff
@@ -832,9 +832,9 @@ void hardswish_backward_kernel(TensorIterator& iter) {
     cpu_kernel_vec(
       iter,
       [&](scalar_t grad_val, scalar_t self_val) -> scalar_t {
-        if (float(self_val) < neg_three) {
+        if (float(self_val) <= neg_three) {
           return zero;
-        } else if (float(self_val) <= three) {
+        } else if (float(self_val) < three) {
           return float(grad_val) * ((float(self_val) / three) + one_half);
         } else {
           return grad_val;
```

Comment:
If the target is to return a min-norm subgradient, then at `3` the grad should be either the left or the right limit depending on which is smaller in absolute value.

Reply:
Yes, though in this case at 3, we always just use the right-hand limit since 1 < 1.5

Reply:
Ah, yes, the function is continuous,  but it is not continuously differentiable,  my bad!